std/digest/package.d

```diff
@@ -259,13 +259,12 @@ version(ExampleDigest)
 @system unittest
 {
     //Using the OutputRange feature
-    import std.algorithm.mutation : copy;
     import std.digest.md;
-    import std.range : repeat;
+    import std.range : repeat, put;
 
     auto oneMillionRange = repeat!ubyte(cast(ubyte)'a', 1000000);
     auto ctx = makeDigest!MD5();
-    copy(oneMillionRange, &ctx); //Note: You must pass a pointer to copy!
+    put(ctx, oneMillionRange);
```

Comment:
Can revert this, and remove the `&`. Leave the note, but just show how `ctx` must be an lvalue for this to work.